api/airgo/model.go

```diff
@@ -0,0 +1,51 @@
+package airgo
+
+import "github.com/XrayR-project/XrayR/api"
+
+type NodeInfoResponse struct {
+	ID             int64  `json:"id"`
+	NodeSpeedlimit int64  `json:"node_speedlimit"`
+	TrafficRate    int64  `json:"traffic_rate"`
+	NodeType       string `json:"node_type"`
+	Remarks        string `json:"remarks"`
+	Address        string `json:"address"`
+	Port           int64  `json:"port"`
+	Scy            string `json:"scy"`
```

Comment:
The key shoule have some comment.

Reply:
Why are other panels not needed?

Reply:
I don't know what the `scy` 's meaning. Its not a standard word.